Source/Particles/ParticleBoundaries_K.H

```diff
@@ -20,21 +20,43 @@ namespace ApplyParticleBoundaries {
     void
     apply_boundary (amrex::ParticleReal& x, amrex::Real xmin, amrex::Real xmax,
                     bool& change_sign_ux, bool& particle_lost,
-                    ParticleBoundaryType xmin_bc, ParticleBoundaryType xmax_bc)
+                    ParticleBoundaryType xmin_bc, ParticleBoundaryType xmax_bc,
+                    amrex::Real refl_xmin, amrex::Real refl_xmax,
```

Comment:
Not that it's immediately important, but `refl_x{min,max}` could probably be better named `refl_probability_x{min.max}`